src/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationAsyncResult.cs

```diff
@@ -150,8 +150,14 @@ internal bool CancelStreamOperation()
             if (_callbackInvoked)
                 return false;
 
-            _asyncStreamOperation.Cancel();
-            return true;
+			try
+            {
```

Comment:
Nit: the spacing is off in the code... looks like a mix of tabs/spaces were used. Tabs shouldn't be used.